zendesk/deco
# Add performance Tests and Health Checks

:potable_water:

Add some documentation related to the Finatra [HTTP Admin interface](https://twitter.github.io/twitter-server/Admin.html#admin-interface). With this interface, we can run convenient and comprehensive checks on diagnostics, profiling, and much more.

This code change also enables performance testing with `jmeter`, an added tool inspired by the [pravda project](https://github.com/zendesk/pravda).

/cc @zendesk/sylvius

### Reference:
JIRA: https://zendesk.atlassian.net/browse/TR-51

### How could this be manually tested?
* Make sure the server is running: `sbt run`
* Visit `http://localhost:9990/admin`
* Install jmeter: `brew install jmeter`
* Run jmeter: `jmeter`
* Load the `DecoSummaryReport.jmx` file and run the Jmeter tests (See updated README for more details)

### Risks
* None
